index.html

```diff
@@ -1,192 +1,202 @@
+                            <i class="arrow down"></i>
+                    </div>
+                    <ul class="language-description" ;">
+                        <li class="skills-parent">
+                            <img class="language-icon" src="img/js.png" alt="javascript">
+                            <p class="name">Javascript</p>
+                        </li>
+                        <li class="skills-parent">
+                            <img class="language-icon" src="img/html.png" alt="html">
+                            <p class="name">HTML</p>
+                        </li>
+                        <li class="skills-parent">
+                            <img class="language-icon" src="img/css.png" alt="css">
+                            <p class="name">CSS</p>
+                        </li>
+                    </ul>
+                    </li>
+                    <li class="languages separator">
+                        <h3 class="language-title">Frameworks</h3>
+                        <i class="arrow right"></i>
+                    </li>
+                    <li class="languages separator">
+                        <h3 class="language-title">Skills</h3>
+                        <i class="arrow right"></i>
+                    </li>
+                    <li class="languages separator">
+                        <h3 class="language-title"></h3>
+                    </li>
                 </ul>
-                <a href="#" class="preview">See Project</a>
             </div>
-        </article>
-        <section class="About-container">
-        <div class="about">
-            <h2 class="title-about">About </br> Myself</h2>
-            <p class="text">Hello I’m a software developer! I can help you build a product , feature or website Look through
-                some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate
-                to contact me.</p>
-            <h3 class="connect-about">LET'S CONNECT</h3>
-            <ul class="icon-about">
-                <li><a href="https://github.com/amiralda"><img src="img/github.svg" alt="github-icon"></a></li>
-                <li><a href="https://www.linkedin.com/danyaugustin/"><img src="img/linkedin.svg" alt="linkedin-icon"></a></li>
-                <li><a href="https://angel.co/u/dany-augustin"><img src="img/angellist.svg" alt="angellist-icon"></a></li>
-                <li><a href="www.twitter.com/amiralda"><img src="img/twitter.svg" alt="twitter-icon"></a></li>
-                <li><a href="https://www.medium.com/amiralda/"><img src="img/medium.svg" alt="medium-icon"></a></li>
-            </ul>
-            <a href="#" class="preview">Get my resume</a>
-        </div>
-        <ul class="skills">
-            <div>
-                <li class="languages">
-                    <h3 class="language-title">Languages</h3>
-                    <i class="arrow down"></i>
+        </section>
+        <section class="contact">
```

Comment:
- [Optional] Kinldy fix your contact section on the mobile version. Your mobile version should look like the figma template even after implementing the desktop version.
### Your design
![image](https://user-images.githubusercontent.com/94551484/236610988-b7ca0c84-435f-4b9c-9836-2d2193307e37.png)
### figma
![image](https://user-images.githubusercontent.com/94551484/236611011-3d640a67-0780-4203-9220-b2446faf5ddc.png)